app/Models/product_order.php

```diff
@@ -5,7 +5,7 @@
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 
-class product_order extends Model
+class Product_Order extends Model
```

Comment:
Class names are UpperCamelCase
```suggestion
class ProductOrder extends Model
```